techniques/system/server-common/1.0/reloadRudderServices.st

```diff
@@ -45,6 +45,8 @@ bundle agent system_reload_rudder_services {
       "any" usebundle => service_restart("${jetty_service_name}");
     rudder_server_system_restart_apache::
       "any" usebundle => service_reload("${apache_service_name}");
+    rudder_server_system_reload_relayd.!rudder_server_system_restart_relayd::
```

Comment:
You need to add the reload to the prefixes list to have a non lying reporting.

Reply:
indeed